core/chainlink.Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN make yarndep
 
 COPY contracts ./contracts
 COPY tsconfig.cjs.json ./
+COPY core core
```

Comment:
We copy `core` here because the GQL schemas are stored here and we need them to build `operator ui`

Reply:
Would it be possible/beneficial to be more precise here than all of core?

Reply:
Good idea. Yeah that should be doable